packages/zowe-explorer/README.md

```diff
@@ -23,31 +23,28 @@ More information:
 
 ## Contents
 
-- [What's new in Zowe Explorer 1.18.0](#whats-new-in-zowe-explorer-1.18.0)
+- [What's new in Zowe Explorer 1.18.1](#whats-new-in-zowe-explorer-1181)
 - [Prerequisites](#prerequisites)
 - [Getting Started](#getting-started)
 - [Sample Use Cases](#sample-use-cases)
 - [Credentials Security](#credentials-security)
 - [Usage Tips](#usage-tips)
 - [Extending Zowe Explorer](#extending-zowe-explorer)
 
-## What's new in Zowe Explorer 1.18.0
+## What's new in Zowe Explorer 1.18.1
 
 **Added**:
 
-- Added the ability to register custom profile types in `ProfilesCache` for extenders.
-- Added the ability to pass account and other information from tso profile.
-- Add profiles cache to extenders.
+- Added a reminder to the PR template to create a PR from `master` to `next` after PR is merged into `master` branch.
+- Added @zowe/cli dependency to API.
```

Comment:
should we combine all hot fixes in the "what's new" in the minor level version?
maybe something like What's new in ZE 1.18 (and leave it at that)
and either have a small subsection for hotfixes or prefix the hot-fix change-lines with  `Patch:`

Reply:
We did this last time and then we change it to add the 3rd number. Maybe we can discuss this on stand-up. I kinda like the 1.18.0 / 1.18.1 format

Reply:
@zFernand0 I think this is a good idea to consider for next time. Our Readme changelog is currently separated into Added/Changed/Fixed sections, so I'm not quite sure how the patch would fit into this formatting. We might want to discuss the proposal with @IgorCATech. For this release, users can still get to the 1.18.0 changes from the Changelog file (which is also linked from the Readme).